api/bin/chainflip-ingress-egress-tracker/Cargo.toml

```diff
@@ -31,8 +32,10 @@ chainflip-engine = { path = "../../../engine/" }
 utilities = { path = "../../../utilities" }
 cf-primitives = { path = "../../../state-chain/primitives" }
 pallet-cf-environment = { path = "../../../state-chain/pallets/cf-environment" }
+pallet-cf-ingress-egress = { path = "../../../state-chain/pallets/cf-ingress-egress" }
+pallet-cf-broadcast = { path = "../../../state-chain/pallets/cf-broadcast" }
 state-chain-runtime = { path = "../../../state-chain/runtime" }
 cf-chains = { path = "../../../state-chain/chains" }
 
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/chainflip-io/substrate.git", tag = 'chainflip-monthly-2023-08+3' }
\ No newline at end of file
+substrate-build-script-utils = { git = "https://github.com/chainflip-io/substrate.git", tag = 'chainflip-monthly-2023-08+3' }
```

Comment:
We're actually at `+5` now, maybe this was missed when that was done for all the other substrate deps.